engines/bops_api/app/controllers/bops_api/v2/public/planning_applications_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module V2
+    module Public
+      class PlanningApplicationsController < PublicController
```

Comment:
On reflection, I wonder if we should just inherit from `ApplicationController` here and only add a `PublicController` if we find some code needs to be shared between controllers?

On the other hand, perhaps it's a nice way to indicate intent to make something public: we'll know `ApplicationController` is a mistake and everything should always be explicitly `Public` or `Authenticated`.

Reply:
Yeah it's not needed but inclined to just leave it in for now as I agree with your second point. Although I am struggling to think what would be explicit to this controller